client/src/components/LoginPage/LoginPage.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { createMemoryHistory } from "history";
+import { Router } from "react-router-dom";
+import ShallowRenderer from "react-test-renderer/shallow";
+
+import LoginPage from ".";
+
+let component;
+
+beforeEach(() => {
+    component = shallow(<LoginPage />);
+});
+
+test("it renders correctly", () => {
+    const shallowRenderer = new ShallowRenderer();
+    const snapshotComponent = shallowRenderer.render(<LoginPage />);
+    expect(snapshotComponent).toMatchSnapshot();
+});
+
+test("Route changes to '/path' when sign up clicked", () => {
```

Comment:
I am slightly confused by this test because the name doesn't seem to match the test i.e. `/path` versus `/` ?

Reply:
Whoops, this is a placeholder that I forgot to make match the actual route name!